src/bioutils/sequences.py

```diff
@@ -441,7 +453,12 @@ def translate_cds(seq, full_codons=True, ter_symbol="*"):
     protein_seq = list()
     for i in range(0, len(seq) - len(seq) % 3, 3):
         try:
-            aa = dna_to_aa1_lut[seq[i:i + 3]]
+            the_seq = seq[i:i + 3]
+            wildcard_nucleotides = ["B", "D", "H", "V", "N", "U", "W", "S", "M", "K", "R", "Y", "Z"]
```

Comment:
IUPAC calls these "ambiguity codes". Please use that name so that the intent is clearer.
(e.g., something like `iupac_ambiguity_codes = "BDHVNUWSMKRYZ"`)

Also, a list of chars is better written as a string for readability. (Lists and strings are both [Sequences](https://docs.python.org/3/glossary.html#term-sequence) and have the same interface for lookup, length, iterability, etc.)